src/Tooltip.tsx

```diff
@@ -7,6 +7,7 @@ import {ComponentProps} from './utils/types'
 
 const TooltipBase = styled.span<SxProp>`
   position: relative;
+  display: inline-block;
```

Comment:
We have `display:inline-block` on the pseudo elements of the tooltip ([ref](https://github.com/primer/react/blob/main/src/Tooltip.tsx#L63)) and I wonder if it was working before and got broken after some changes maybe on css? 🤷🏻‍♀️

Reply:
I feel a bit worried that we don't know how/when this broke, because I'm not sure if this will break something for dotcom 😅

Seems like the example in the docs has been broken since 35.2.x 🤔 https://primer-d4a57fc365-13348165.drafts.github.io/Tooltip

Reply:
One idea for our test suite could be to add in an e2e test for this and snapshot the story with the button focused. Definitely would not help with upstream testing, though, if that's what we're looking for.

Reply:
I think e2e test is a great idea - at least we can catch thing in our side. I'll add that. No idea what to do catch it at dotcom though. @langermank brought up VRT tests at dotcom topic a few weeks ago to the eng sync and this might be something we can rely on in the (hopefully near) future!